vault/audit.go

```diff
@@ -747,6 +749,11 @@ func (c *Core) handleAuditLogSetup(ctx context.Context) error {
 }
 
 func (c *Core) addAuditFromConfig(ctx context.Context, auditConfig *server.AuditDevice) error {
+	if c.standby {
+		c.logger.Warn("audit device present in standby but not active node", "path", auditConfig.Path)
+		return nil
+	}
+
```

Comment:
I guess this will led to false positives when you have just added a new audit device and are syncing the config across your nodes, but that might be fine.

Also, I had a hard time understanding the sentence. I think we should add a little more details to make it easier for end-users to understand.

```suggestion
	if c.standby {
		c.logger.Warn("audit device present in local configuration but not in the configuration of the active node", "path", auditConfig.Path)
		return nil
	}

```